mlir/test/Examples/nvgpu/lit.local.cfg

```diff
@@ -0,0 +1,3 @@
+config.unsupported = False
+if not config.enable_cuda_runner or not config.mlir_run_cuda_sm90_tests:
```

Comment:
I think this is important that its sm90 tests (see discourse thread about using these with non-tensor cores, I think the expectations about behavior, esp fallback, seems missing)

Reply:
where is that thread?